root/etc/plex/plex-update

```diff
@@ -32,20 +32,12 @@ else
   installedVersion="none"
 fi
 
-# Read set version
-readVarFromConf "version" versionToInstall
-if [ -z "${versionToInstall}" ]; then
-  echo "No version specified in install.  Broken image"
-  exit 1
-fi
-
-# Short-circuit test of version before remote check to see if it's already installed.
-if [ "${versionToInstall}" = "${installedVersion}" ]; then
-  exit 0
-fi
```

Comment:
Ok, this is a good call out. I was tripped up by using the version as a TAG and having TAG also be a build-arg. 

I switched things up a bit: now, you can build the image with TAG=autoupdate (this obsoletes beta/public - but you can still build the images with public/beta tags - it will behave as if it was 'autoupdate' but it will ignore the AUTO_UPDATE_CHANNEL and will be hardcoded to 'beta' or 'public').